.changes/api-convert-file-src-mock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@tauri-apps/api': 'minor:feat'
```

Comment:
@amrbashir can we change this to a patch so we can release the Debian package hotfix?

Reply:
we've got to have selective release in our workflow, this is not scalable IMO, but I will revert this bump.

Reply:
https://github.com/tauri-apps/tauri/pull/8033

Reply:
yeah we should have that :( it's always so boring.